kimchi/src/lagrange_basis_evaluations.rs

```diff
@@ -2,18 +2,85 @@ use ark_ff::{batch_inversion_and_mul, FftField};
 use ark_poly::{EvaluationDomain, Evaluations, Radix2EvaluationDomain as D};
 use rayon::prelude::*;
 
-/// The evaluations of all normalized lagrange basis polynomials at a given
-/// point. Can be used to evaluate an `Evaluations` form polynomial at that point.
+/// Evaluations of all normalized lagrange basis polynomials at a given point.
+/// Can be used to evaluate an `Evaluations` form polynomial at that point.
+///
+/// The Lagrange basis for polynomials of degree <= d over a domain
+/// {ω_0,...,ω_d} is the set of (d+1) polynomials ${l_0,...,l_d}$ of degree d
+/// that equal 1 at ω_i and 0 in the rest of the domain terms. They can be used
+/// to evaluate polynomials in evaluation form efficiently in O(d) time.
+///
+/// When chunking is in place, the domain size is larger than the maximum
+/// polynomial degree allowed. In particular, the SRS is not long enough to
```

Comment:
```suggestion
/// polynomial degree allowed. In particular, the SRS (used by the polynomial commitment to commit to) is not long enough to
```

or something like this, to just remind that the SRS (nit - we should stop using the term SRS as it is always "structured") is used by the PCS, and the domain size is related to the field we use to evaluate the polynomials at - the evaluation domain size is unrelated to the SRS size.